digitalpy/core/main/state_change_event.py

```diff
@@ -9,7 +9,7 @@
 #######################################################
 from typing import Any
 from digitalpy.core.persistence.persistent_object import PersistentObject
-from digitalpy.core.event import Event
+from digitalpy.core.main.event import Event
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position